docs/tutorials/setup_acquisition/select.md

```diff
@@ -31,9 +31,12 @@ Output of the above code is below:
 <DeviceIdentifier Storage "Zarr">
 <DeviceIdentifier Storage "ZarrBlosc1ZstdByteShuffle">
 <DeviceIdentifier Storage "ZarrBlosc1Lz4ByteShuffle">
+<DeviceIdentifier Storage "ZarrV3">
+<DeviceIdentifier Storage "ZarrV3Blosc1ZstdByteShuffle">
+<DeviceIdentifier Storage "ZarrV3Blosc1Lz4ByteShuffle">
 ```
 
-All identified devices will be listed, and in the case of this tutorial, no cameras were connected to the machine, so only simulated cameras were found. Note that discovered storage devices will also print.
+All identified devices will be listed, and in the case of this tutorial, no camera drivers were installed on the machine, so only simulated cameras were found. Note that discovered storage devices will also print.
```

Comment:
All the (Acquire) camera drivers should be installed, but device discovery depends on the vendor's drivers being installed and the cameras being properly connected.